core/kazoo_json/test/kz_json_test.erl

```diff
@@ -55,18 +56,15 @@ prop_get_value() ->
                       end)
            ).
 
--define(DO_NOT_RUN_QC_SET_VALUE, true).
--ifndef(DO_NOT_RUN_QC_SET_VALUE).
```

Comment:
I have yet to see it fail with these changes. Disabling tests is wrong and bad and you should feel bad for doing it! :)